### ISSUE TITLE
Error in the `getMetadata` function (BFC issue?)

### ISSUE THREAD

**Issue by shbrief** [shbrief/OmicsMLRepoR#22]
Multiple BFC? I got the error which seems to come from this line:
https://github.com/shbrief/OmicsMLRepoR/blob/1efd0f9bf015a4e16a1eccdc5b6a37d4ad6462a7/R/getMetadata.R#L20

Error:
```
> cmd <- getMetadata("cMD")
Error in if (bfcneedsupdate(bfc, rid)) { : the condition has length > 1
```

Possible issue:
```
> rname <- "https://storage.googleapis.com/omics_ml_repo/cMD_curated_metadata_release.csv"
> bfc <- s2p_get_cache()
> bfcres <- bfcquery(bfc, rname, 'rname')
> rid <- bfcres$rid
>
> bfc
class: BiocFileCache
bfccache: /mnt/STORE1/bighome/sehyun/.cache/R/OmicsMLRepo
bfccount: 5
For more information see: bfcinfo() or bfcquery()
> bfcneedsupdate(bfc, rid)
 BFC1  BFC2  BFC4  BFC6  BFC8 
 TRUE FALSE FALSE FALSE FALSE 
```

**Comment by shbrief** [comment]
@seandavi Hi Sean, this function (`getMetadata()`) is based on the `GenomicSuperSignature::getModel()` you wrote. I encounter an error and wonder whether you can take a look at it. Thanks!

**Comment by shbrief** [comment]
Merged to #26
